test/types-multipart.spec.js

```diff
@@ -229,6 +229,25 @@ describe('types-multipart', () => {
       ],
       what: 'Empty content-type and empty content-disposition'
     },
+    {
+      config: {
+        isPartAFile: (fieldName) => (fieldName !== 'upload_file_0')
```

Comment:
You might add a test for the other way around where a part that originally would be treated as `field` now will be treated as `file`.

Reply:
done